### ISSUE TITLE
The social media icons from the "Share" panel cannot be focused using the keyboard

### ISSUE THREAD

**Issue by Softvision-CristinaBadescu** [mozilla-services/screenshots#4412]
[Affected versions]:
- Firefox 60.0 and up

[Affected Platforms]:
- All Windows
- All Mac
- All Linux

[Prerequisites]:
- Have a saved shot link.

[Steps to reproduce]:
1. Open the Firefox browser with the profile from prerequisites and navigate to the saved shot link.
2. Click the "Share" button.
3. Press the "Tab" key once and observe the behavior.

[Expected result]:
- The "Facebook" icon is focused.

[Actual results]:
- The "Email" icon is focused.

[Notes]:
- Only the email, shareable link field and the "Copy" button can be focused using the keyboard.
- Here is a screen recording with the issue:
![ss - share social media icons](https://user-images.githubusercontent.com/31244235/39861867-a8cc5282-544a-11e8-8ec4-37eefcf0faae.gif)

**Comment by punamdahiya** [comment]
With new UI, share button is not shown on shot page.
